src/redux/StudentEdit.redux/StudentEdit.reducer.js

```diff
@@ -0,0 +1,35 @@
+import {
+  EDIT_USER_START,
+  EDIT_USER_SUCCESS,
+  EDIT_USER_FAILURE,
+} from './StudentEdit.action';
+
+const userEditInitialState = {
+  editingUser: [],
```

Comment:
I don't think we need this variable. Unless I'm mistaken about what it's for, we're going to keep the form state local to StudentInfo.js

Reply:
removed variable